### DISCUSSION TITLE
osu! not minimizing like stable release

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#18385] [Q&A]
I am a streamer and use OBS to capture osu! while I play. I run the game on full screen but with a lower resolution than my monitor because I have a huge monitor and can't play osu! on it like I can when the window is a bit smaller. I have a monitor source under a game capture source, so viewers can still see my monitor screen when I tab out of osu!. Here is a video of this happening. https://streamable.com/s8f056
The top left of the video is just a monitor source on its own and the bottom right is a game capture source (osu!) and a monitor underneath.

On osu! Lazer, I tried doing the same thing and changed the scale of the Lazer window to 75% horizontal and vertical so it achieves the same effect for playing like osu! stable. I found it when I have the game capture source and the monitor source underneath for osu! Lazer, the Lazer window just freezes and doesn't disappear, it only freezes the Lazer window and doesn't disappear to reveal the monitor source. Here is a video of this happening. https://streamable.com/o4btw6
The top left of the video is just a monitor source on its own and the bottom right is a game capture source (osu!) and a monitor underneath.

I noticed that sometimes the osu! Stable game capture source won't minimize if I have the "Confine mouse cursor:" set to "Never", so I thought this could maybe be an issue that can be fixed in Lazer.

Ultimately if anyone knows how I can fix this in osu! Lazer or OBS I would greatly appreciate it!

**Comment by peppy** [comment]
You should probably use two scenes. It sounds like you are relying on undefined behaviour in your drivers / OBS / osu! which I don't think we can ever guarantee.

**Reply by ghost** [reply]
I see. Would there be any way to fix this though so I don't have to use two scenes? As it's annoying to switch all the time.

**Comment by Susko3** [comment]
You can use the OBS automatic scene switcher (from the menu bar &rarr; Tools &rarr; automatic scene switcher) instead of relying on the window capture being blank.

Have two scenes and switch depending on what window is active.
